site/app/controllers/student/TeamController.php

```diff
@@ -3,42 +3,44 @@
 namespace app\controllers\student;
 
 use app\controllers\AbstractController;
-use app\libraries\Core;
 use app\libraries\FileUtils;
+use Symfony\Component\Routing\Annotation\Route;
 
 class TeamController extends AbstractController {
     public function run() {
         switch ($_REQUEST['action']) {
             case 'create_new_team':
-                $this->createNewTeam();
+                $this->createNewTeam($_REQUEST['gradeable_id']);
```

Comment:
Should you be passing in `$_REQUEST['gradeable_id']` into the function calls? What would happen if `$_REQUEST['gradeable']` is null or the gradeable_id passed through the request is invalid?
I think it would be safer to handle this inside of each function and return the appropriate response unless there's an explicit reason why this is set up the way it is.

Reply:
This is for backward compatibility to avoid breaking existing testing code if they rely on `run`. `run` is not used in actual routing process. Hmm it's actually safe to remove them completely.